src/connection.service.ts

```diff
@@ -87,13 +87,21 @@ export class ConnectionService {
         this.serviceOptions.heartbeatInterval
       ).subscribe(() => {
         if (this.currentState.hasNetworkConnection) {
-          fetch(this.serviceOptions.heartbeatUrl, { mode: "no-cors" }).then(
-            (response) => {
+          const timestamp = new Date().getTime();
+          fetch(`${this.serviceOptions.heartbeatUrl}?_=${timestamp}`)
+            .then((response) => {
               this.currentState.hasInternetAccess =
                 response.ok || response.type === "opaque";
               this.emitEvent();
-            }
-          );
+            })
+            .catch((e) => {
+              this.currentState.hasInternetAccess = false;
+              this.emitEvent();
+              console.log(
```

Comment:
console.error would probably be better in this case, unless you want the log to be low key